--- a/src/codegen/bundle-modules.ts	2026-08-20 02:50:33.000000000 +0200
+++ b/src/codegen/bundle-modules.ts	2026-08-25 07:43:40.232127211 +0200
@@ -260,6 +260,9 @@
       .replace(/import.meta.require\((.*?)\)/g, (expr, specifier) => {
         throw new Error(`Builtin Bundler: do not use import.meta.require() (in ${file_path}))`);
       })
+      // esbuild --format=esm emits export default require_X(); bun's bundler
+      // would have turned that into the IIFE return value.
+      .replace(/export default /g, "return ")
       .replace(/return \$\nexport /, "return")
       .replace(/__intrinsic__/g, "@")
       .replace(/__no_intrinsic__/g, "") + "\n";
